internal/db/db_blueprints.go

```diff
@@ -53,15 +53,28 @@ const (
 	sqlGetBlueprints = `
 		SELECT blueprints.id, blueprints.name, blueprints.description, MAX(blueprint_versions.version) as version, MAX(blueprint_versions.created_at) as last_modified_at
 		FROM blueprints INNER JOIN blueprint_versions ON blueprint_versions.blueprint_id = blueprints.id
-		WHERE blueprints.org_id = $1 AND blueprints.account_number = $2
+		WHERE blueprints.org_id = $1
 		GROUP BY blueprints.id
 		ORDER BY last_modified_at DESC
-		LIMIT $3 OFFSET $4`
+		LIMIT $2 OFFSET $3`
+
+	sqlFindBlueprints = `
+		SELECT blueprints.id, blueprints.name, blueprints.description, MAX(blueprint_versions.version) as version, MAX(blueprint_versions.created_at) as last_modified_at
+		FROM blueprints INNER JOIN blueprint_versions ON blueprint_versions.blueprint_id = blueprints.id
+		WHERE blueprints.org_id = $1 AND (blueprints.name ILIKE $4 OR blueprints.description ILIKE $4)
```

Comment:
```suggestion
		WHERE blueprints.org_id = $1 AND (blueprints.name ILIKE $4 AND blueprints.description ILIKE $4)
```

Not sure but this would be more helpful, I think. If I inserted too general name, then the description would not really filter out the result.

Reply:
But we want to filter blueprints by name or by description. no?

Reply:
Oh, I see now. Yeah, we have the same argument $4, then this makes sense. :)